### PR TITLE
fixed typo in propertyy params in TestReults.ps1

### PR DESCRIPTION
**Pull request by stan-alam**
## PR Checklist

- [ ] PR has meaningful title
- [ ] Summary describes changes
- [ ] PR is ready to be merged
  - If not, use the arrow next to `Create Pull Request` to mark it as a draft. PR can be marked `Ready for review` when it's ready.
- [ ] Tests are added/update *(if required)*
- [ ] Documentation is updated/added *(if required)*

### CODE REVIEW

**Comment by fflaten** [comment]
Thanks for catching and fixing this 🙌

File: src/functions/TestResults.ps1 [hunk 1]
```diff
@@ -63,7 +63,7 @@ function Export-NUnitReport {
     -Passthru or by using the Run.PassThru configuration-option.
 
     .PARAMETER Path
-    The path where the XML-report should  to the ou the XML report as string.
+    The path where the XML-report should output the XML report as string.
```
Comment (fflaten) [inline src/functions/TestResults.ps1:66]:
```suggestion
    The path where the XML-report should be saved.
```

Is this better? The original sentence wasn't great. 🙂

File: src/functions/TestResults.ps1 [hunk 2]
```diff
@@ -116,7 +116,7 @@ function Export-JUnitReport {
...
```
Comment (fflaten) [inline src/functions/TestResults.ps1:119]:
```suggestion
    The path where the XML-report should be saved.
```

See previous comment

**Review by fflaten — COMMENTED** [review]
Thanks for catching and fixing this. Should we rewrite the sentence completely?

File: src/functions/TestResults.ps1 (hunk 2 above)
Comment (stan-alam) [reply src/functions/TestResults.ps1:119]: great suggestion!  "The path where the XML-report should be saved.

**Pull request merged**